websocket/_http.py

```diff
@@ -88,6 +88,8 @@ def _start_proxied_socket(url: str, options, proxy) -> tuple:
 
     hostname, port, resource, is_secure = parse_url(url)
 
+    proxy_type = None
+    rdns = False
```

Comment:
This line is already found inside the first if statement and in the sock5 elif. Should it be removed from those places to avoid duplication?